circleindicator/src/main/java/me/relex/circleindicator/OnIndicatorClickListener.java

```diff
@@ -0,0 +1,8 @@
+package me.relex.circleindicator;
+
+/**
+ * Created by Tobias.Sasse on 11/03/2016.
+ */
```

Comment:
I think it is better to remove this kind of comment because in the future it won't be updated